### PR TITLE
[FEATURE] Adding Insertion / Deletion / Similarity / Identity to AlignmentStats

### PR DESCRIPTION
**Pull request by bnbowman**
As discussed on the seqan-dev mailing-list here:
https://lists.fu-berlin.de/pipermail/seqan-dev/2015-January/msg00006.html

### CODE REVIEW

**Comment by esiragusa** [comment]
@bnbowman Thanks for the PR!

@rrahn This change should go into develop only.

File: core/include/seqan/align/evaluate_alignment.h [hunk 1]
```diff
@@ -240,6 +266,10 @@ TScoreVal computeAlignmentStats(AlignmentStats & stats,
     SEQAN_ASSERT(it0 == itEnd0);
     SEQAN_ASSERT(it1 == itEnd1);
 
+    // Finally, compute the alignment similarity from the various counts
+    stats.alignmentSimilarity = 100.0 * (float)stats.numPositiveScores / (float)length(row(align, 0));
+    stats.alignmentIdentity = 100.0 * (float)stats.numMatches / (float)length(row(align, 0));
```
Comment (rrahn) [inline core/include/seqan/align/evaluate_alignment.h]: Please, use static_cast<float> instead of C-style cast.

**Comment by rrahn** [comment]
Please update the minor remark and we can push it to develop. 
Otherwise it looks good to me!

**Comment by bnbowman** [comment]
Done.

**Comment by rrahn** [comment]
This PR is replaced by #796 as we only put this into develop which is then present in the new release.

**Pull request closed without merging**